appveyor.yml

```diff
@@ -1,7 +1,7 @@
-version: 0.1.0.{build}
-configuration: Release 
+version: '{branch}-{build}'
+configuration: Debug 
 platform: Any CPU
-
+image: Visual Studio 2017
```

Comment:
Do we need to have the secret in line 7 checked in?

Reply:
This is not our secret. I encrypt config file (containing sensitive info like password) and commit to github. 
Line 7 is "secure variable" in appveyor for this CI to decrypt our config files. 
Details found in: https://www.appveyor.com/docs/how-to/secure-files/